content/docs/command-reference/exp/resume.md

```diff
@@ -0,0 +1,48 @@
+# exp resume
+
+Resume (continue) existing checkpoint experiments.
```

Comment:
This ended up as `exp run --resume` right? Is there a PR or explanation of the differences? That seems outdated in https://github.com/iterative/dvc/wiki/Experiments too. Feel free to suggest in #2200 (options list of `exp run`).